.dockerignore

```diff
@@ -0,0 +1,2 @@
+kubevirtci
+*/**/go*
```

Comment:
This would prevent cloning of any random module that starts with "go". e.g. gotohell.go. Can we lock it more specifically?